faqs/Tech/what-is-musd.mdx

```diff
@@ -0,0 +1,38 @@
+---
+id: what-is-musd
+sidebar_position: 2
+title: What is MUSD?
+---
+
+import CustomReactPlayer from "../../src/components/CustomReactPlayer";
+import RoundingError from "@site/static/img/faqs/roundingerror.png";
+
+MUSD is a stablecoin that allows users to borrow from the Moria Treasury, a contract on the Bitcoin Cash network. Users must provide, at a minimum 150%, collateral in BCH. MUSD is built on the Moria Protocol.
+
+The Moria Protocol is a decentralized stablecoin protocol built on top of the [CashTokens](what-is-cashtokens) system on Bitcoin Cash. It's currently being used to allow users to borrow the MUSD stablecoin. You can check out the [Moria Protocol White Paper Here.](/Moria-Whitepaper.pdf)
+
+## What is a stablecoin?
+All stablecoins are not equal. A stablecoin is a token that is tied to any kind of asset. That could be the USD; a stock; gold, really, just about anything. How that token is tied to said asset can vary from a centralized solution by means through a company, algorithmically, or with market forces. MUSD is a decentralized, floating peg stablecoin. That floating peg aims towards parity via market forces. Because the contract during loan creation is a fixed 1:1 (USD to MUSD) ratio, market forces will try to keep that peg very close to that ratio.
+
+## What's the difference between Tether and MUSD?
+[Tether, also known by its ticker USDT](/faqs/Scams/tether), is a type of stablecoin. There haven't been any full audits of Tether's reserves to verify that there are enough Dollars to match Tether's supply. MUSD is completely transparent and is heavily overcollateralized. [Check out the stats.](https://app.moria.money/stats)
+
+## Potential issues
+MUSD is in alpha mode, you should expect bugs and potential loss of funds. From [Moria's site](https://docs.riftenlabs.com/moria/testrun/), it reads:
```

Comment:
Instead of "From Moria's site", write "It is very important to read the disclaimers & risks section on the Moria website" and link from the "disclaimers & risks part".  Be descriptive about the link.

Reply:
Fixed.